x-pack/plugins/case/server/routes/api/cases/sub_case/get_sub_case.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { SubCaseResponseRt } from '../../../../../common/api';
+import { RouteDeps } from '../../types';
+import { flattenSubCaseSavedObject, wrapError } from '../../utils';
+import { SUB_CASE_DETAILS_URL } from '../../../../../common/constants';
+import { countAlertsForID } from '../../../../common';
+
+export function initGetSubCaseApi({ caseService, router }: RouteDeps) {
+  router.get(
+    {
+      path: SUB_CASE_DETAILS_URL,
+      validate: {
+        params: schema.object({
+          case_id: schema.string(),
+          sub_case_id: schema.string(),
+        }),
+        query: schema.object({
+          includeComments: schema.boolean({ defaultValue: true }),
+        }),
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const client = context.core.savedObjects.client;
+        const includeComments = request.query.includeComments;
+
+        const subCase = await caseService.getSubCase({
```

Comment:
We should check if the subcase is child of the `case_id`. As it is we return a subcase even if it is not a child of the requested case. If we want this functionality we should create another route like `api/subcases/{id}`.